Classes/Public/Manager/Objects/OCTMessageCall.h

```diff
@@ -6,19 +6,19 @@
 //  Copyright (c) 2015 dvor. All rights reserved.
 //
 
-#import "OCTMessageAbstract.h"
+#import "OCTObject.h"
 #import "OCTManagerConstants.h"
 
-@interface OCTMessageCall : OCTMessageAbstract
+@interface OCTMessageCall : OCTObject
 
 /**
  * The length of the call in seconds.
  **/
-@property (assign, nonatomic, readonly) NSTimeInterval callDuration;
+@property  NSTimeInterval callDuration;
```

Comment:
As OCTCall is in database now, you can add property with call instead of `callDuration`. That way there will be no need in updating duration in message.